debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -377,6 +391,25 @@
     animation: spin 2s linear infinite;
 }
 
+/* Panel and Toolbar hidden button health states */
+#djDebug .djdt-health-1 {
+    background: var(--djdt-health-bg-1) !important;
+    color: var(--djdt-health-color-1) !important;
+}
+
+#djDebug .djdt-health-2 {
+    background: var(--djdt-health-bg-2) !important;
+    color: var(--djdt-health-color-2) !important;
+}
+
+#djDebug .djdt-toolbarhandle-health-1 {
+    border-color: var(--djdt-health-bc-1) !important;
+}
+
+#djDebug .djdt-toolbarhandle-health-2 {
+    border-color: var(--djdt-health-bc-2) !important;
+}
```

Comment:
Can you add a comment why all these `!important` are necessary? And/or remove them when they are not?